### PR TITLE
Add Flashcard UI

### PR DESCRIPTION
**Pull request by islandbitcoin**

### CODE REVIEW

File: app/navigation/root-navigator.tsx [hunk 1]
```diff
@@ -516,6 +519,23 @@ export const ChatNavigator = () => {
   )
 }
 
+const StackCards = createStackNavigator<CardStackParamList>()
+
+export const CardNavigator = () => {
```
Comment (Nodirbek75) [inline app/navigation/root-navigator.tsx:524]: I think you don't need to make different navigation stack for Card screen. You can simply add CardScreen component to the Tab navigator like HomeScreen.

Comment (islandbitcoin) [reply app/navigation/root-navigator.tsx]: ok I removed the extra navigation stack

File: app/screens/card-screen/card.tsx [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+
+  item: {
+    marginHorizontal: 32,
+    marginVertical: 8,
+  },
+
+  itemContainer: {
+    borderRadius: 8,
+    backgroundColor: colors.grey5,
+  },
```
Comment (Nodirbek75) [inline app/screens/card-screen/card.tsx:404]: Remove item, itemContainer styles

File: app/screens/card-screen/card.tsx [hunk 3]
```diff
@@ -0,0 +1,485 @@
...
+    width: "100%",
+    maxWidth: 184,
+  },
+  removeButton: {
+    marginTop: 16,
+    display: "flex",
+    paddingVertical: 96,
+    borderRadius: 12,
+  },
+}))
```
Comment (Nodirbek75) [inline app/screens/card-screen/card.tsx:485]: Most of the styles not being used, you can remove them to make the component clean

**Review by Nodirbek75 — CHANGES_REQUESTED** [review]
You can remove not used codes from the component to make it clean and easy to read.

File: app/screens/card-screen/card.tsx (hunk 3 above)
Comment (islandbitcoin) [reply app/screens/card-screen/card.tsx:485]: cleaned up all unused styles

**Review by Nodirbek75 — APPROVED** [review]

**Comment by islandbitcoin** [comment]
@Nodirbek75 I'll wait until yours cards arrive to test it before merging

File: app/components/modal-nfc/modal-nfc-flashcard.tsx [hunk 4]
```diff
@@ -31,6 +31,7 @@ export const ModalNfcFlashcard: React.FC<{
     setIsActive(false)
     if (nfcRegistered) {
       try {
+        await NfcManager.cancelTechnologyRequest()
```
Comment (Nodirbek75) [inline app/components/modal-nfc/modal-nfc-flashcard.tsx:34]: Why are you calling cancelTechnologyRequest() method twice here? I think you can remove one.

**Review by Nodirbek75 — CHANGES_REQUESTED** [review]

File: app/components/modal-nfc/modal-nfc-flashcard.tsx (hunk 4 above)
Comment (islandbitcoin) [reply app/components/modal-nfc/modal-nfc-flashcard.tsx:34]: done

**Review by Nodirbek75 — APPROVED** [review]

**Pull request merged**